app/views/offers/index.html.erb

```diff
@@ -27,7 +27,7 @@
     </h2>
 
     <div class="row">
-      <% @offers.each do |offer| %>
+      <% @offers.first(48).each do |offer| %>
```

Comment:
Random number but the reason is because we otherwise with the meetup data we have to load like 1000 offers at once which takes a while. Maybe we can implement some pagination.

Reply:
Agreed. With that amount of data we would have to implement pagination or lazy loading.